Add Firebase dynamic links

MichaelSheely opened the pull request:
Linked to the official deprecation announcement at https://firebase.google.com/support/dynamic-links-faq, but there's also a post at https://newsletter.pragmaticengineer.com/p/google-shutting-down-firebase-dynamic which might be longer lived.

Let me know if you'd like an update to the latter.

MichaelSheely committed:
Add firebase dynamic links which will be turned down in a little over a year

MichaelSheely committed:
Add punctuation

hrichiksite commented:
As per [Firebase Docs](https://firebase.google.com/support/dynamic-links-faq#will_new_and_existing_links_continue_to_work:~:text=need%20to%20know%3F-,On%20August%2025th%2C%202025%2C%20Firebase%20Dynamic%20Links%20will%20shut%20down.,-All%20links%20served)
> On August 25th, 2025, Firebase Dynamic Links will shut down.

But the date here is November 2024.

MichaelSheely committed:
Update end date for firebase dynamic links.

The initial commit was working from this version of the documentation:
https://web.archive.org/web/20230730021027/https://firebase.google.com/support/dynamic-links-faq

The current documentation now has a precise end date of
August 25th, 2025 which is reflected in the json.

MichaelSheely commented:
> As per [Firebase Docs](https://firebase.google.com/support/dynamic-links-faq#will_new_and_existing_links_continue_to_work:~:text=need%20to%20know%3F-,On%20August%2025th%2C%202025%2C%20Firebase%20Dynamic%20Links%20will%20shut%20down.,-All%20links%20served)
> 
> > On August 25th, 2025, Firebase Dynamic Links will shut down.
> 
> But the date here is November 2024.

Thanks!  At the time I made the pull request the documentation looked like this: https://web.archive.org/web/20230730021027/https://firebase.google.com/support/dynamic-links-faq

It appears a concrete date has now been added which is a bit further out compared to the initial "12 months" estimate.

I have pushed a new commit with the updated date of August 25, 2025.

MichaelSheely commented:
Any concerns with this PR?

MichaelSheely commented:
Would it make sense to merge this?  Any concerns?
